R/epiprocess-data.R

```diff
@@ -66,87 +74,117 @@
 #' Modifications:
```

Comment:
issue: we're lacking sourcing + licensing information for the doctor visits part here

Reply:
Waiting on Roni/Carlyn for more info about how to attribute the doctor-visits source.